tpl/entity_entityconfig.html.twig

```diff
@@ -3,55 +3,78 @@
       {{ __('Support name', 'flyvemdm')  }}
    </td>
    <td>
-      <input name="support_name" value="{{ entityConfig.support_name }}" />
-      {% if entityConfig._support_name == -2 %}
+    {% if entityConfig._support_name == -2 %}
+      <input name="support_name" />
```

Comment:
I'm reviewing in deep to understand why you changed this (https://github.com/flyve-mdm/glpi-plugin/pull/427#issuecomment-382663086) and I found this change is not necessary.

If the value is inherited, then ```entityConfig.support_name``` is an empty string. Then no need to move the <input> tag in the % IF %. Or did I missed something ?